cmd/cloudFoundryDeploy_test.go

```diff
@@ -1019,10 +741,6 @@ func TestCfDeployment(t *testing.T) {
 
 	// TODO: testCfPushDeploymentWithoutVariableSubstitution is already handled above (?)
 
-	// TODO: testCfBlueGreenDeploymentWithVariableSubstitution variable substitution is not handled at the moment (pr pending).
-	// but anyway we should not test the full cycle here, but only that the variables substitution tool is called in the appropriate way.
-	// variable substitution should be tested at the variables substitution tool itself (yaml util)
-
```

Comment:
Whos' message are we removing here?

Reply:
It is from @marcusholl. It looks like variable substitution is relevant to cf_native blue-green deployment type case. So, I do not see any need of having the TODO comment if blue-green itself is gone. Further info from this PR https://github.com/SAP/jenkins-library/pull/1743